docs/fsharp/language-reference/byrefs.md

```diff
@@ -98,7 +98,48 @@ All of these rules together mean that the holder of an `inref` pointer may not m
 
 ### Outref semantics
 
-The purpose of `outref<'T>` is to indicate that the pointer should only be written to. Unexpectedly, `outref<'T>` permits reading the underlying value despite its name. This is for compatibility purposes. Semantically, `outref<'T>` is no different than `byref<'T>`.
+The purpose of `outref<'T>` is to indicate that the pointer should only be written to. Unexpectedly, `outref<'T>` permits reading the underlying value despite its name. This is for compatibility purposes.
+
+Semantically, `outref<'T>` is no different than `byref<'T>`, except for one difference where `outref<'T>` parameters can be implicitly constructed into a tuple with the return value when a tuple-style method can be changed to return a tuple when you only supply non-`outref<'T>` arguments to the method.
+
+```fs
+type C =
+    static member M1(x, y: _ outref) =
+        y <- x
+        true
+    static member M2(x, y: _ byref) =
+        y <- x
+        true
+        
+match C.M1 1 with
+| true, 1 -> printfn "Expected" // Fine
+| _ -> printfn "Never matched"
+
+match C.M2 1 with // error FS0001: This expression was expected to have type ''a * 'a ref' but here has type 'int'
+| true, 1 -> printfn "Expected" // error FS0001: This expression was expected to have type 'bool'but here has type ''a * 'b' 
+| _ -> printfn "Never matched"
+```
+
+Although programming with `outref<'T>` are not normally seen, this eases interoperability with C#'s e.g. `TryParse` methods, so that instead of being forced to utilise `mutable` variables and more lines of code:
```

Comment:
This doesn't have anything to do with _using_ `outref` in your own code. This kind of implicit construction has always been possible before `outref` was ever introduced. I recommend cutting this section out.